zerver/worker/queue_processors.py

```diff
@@ -2,7 +2,11 @@
 from __future__ import absolute_import
 from typing import Any, Callable, Dict, List, Mapping, Optional, cast
 
-from contrib_bots.bot_lib import BotHandlerApi, StateHandler
+import sys
+import os
+sys.path.insert(0, os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../api')))
+
+from bots_api.bot_lib import BotHandlerApi, StateHandler
```

Comment:
I think this is because of a missing `__init__.py` in the `api` directory. @timabbott shouldn't there be one, since we want `api` to be a package?